app/src/main/java/ch/epfl/sdp/db/DataRepository.java

```diff
@@ -0,0 +1,44 @@
+package ch.epfl.sdp.db;
+
+import android.app.Application;
+
+import androidx.lifecycle.LiveData;
+
+import java.util.List;
+
+public class DataRepository {
```

Comment:
I don't exaclty get what is this class

Reply:
I implement like this because this is the way to use Room(SQLite database) which is suggested here https://codelabs.developers.google.com/codelabs/android-room-with-a-view/index.html?index=..%2F..index#0
Basically Repository: Used to manage multiple data sources.